docs/sysadmin-basics/cluster-config.txt

```diff
@@ -168,6 +168,11 @@ To disable telemetry reporting in both the master and the WebUI, start the maste
 setting an environment variable, as with any other configuration option). Disabling telemetry
 reporting will not affect the functionality of Determined in any way.
 
+Separate from the telemetry reporting mentioned above we also provide functionality to use opentelemetry to collect traces. 
```

Comment:
I'd add a section header and a hyperlink (similar to `.. _master-configuration:` below), so this docs can be more easily referred to.